src/java.desktop/windows/classes/sun/awt/windows/WFramePeer.java

```diff
@@ -80,11 +79,8 @@ public void toFront() {
     private native void setMaximizedBounds(int x, int y, int w, int h);
     private native void clearMaximizedBounds();
 
-    @SuppressWarnings("removal")
     private static final boolean keepOnMinimize = "true".equals(
-        AccessController.doPrivileged(
-            new GetPropertyAction(
-            "sun.awt.keepWorkingSetOnMinimize")));
+            System.getProperty("sun.awt.keepWorkingSetOnMinimize"));
```

Comment:
This file has a now-unused import of `GetPropertyAction`.

Reply:
removed